R/config.R

```diff
@@ -11,7 +11,7 @@
 #' @return the tools.yml file
```

Comment:
These fuctions are lacking proper title, description and parameter documentation. Refer to other files on master to see how it is done. This code will not pass the checks. This applies to all functions. Also the return tag is not quite accurate, what returns is the parsed tools.yml file.